### DISCUSSION TITLE
Support for Telescope "find string"?

### DISCUSSION THREAD

**Discussion by CCurl** [LazyVim/LazyVim#1249] [Q&A] [answered]
In telescope, the default keypress sequence for "do a ripgrep for the word under the cursor" is \<leader\>fs. I have been unable to find that in the \<which-key\> for lazy-vim.

Is it there somewhere? Or, how do I add it?

**Comment by dpetka2001** [comment, marked as answer]
`<leader>sw`

**Reply by CCurl** [reply]
Thanks ... just different terminology than what I was looking for, but 100% the same result. :)
